eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -427,6 +427,7 @@
   <suppress checks="LineLength" files="com.azure.ai.textanalytics.implementation.TextAnalyticsClientImplBuilder"/>
   <suppress checks="LineLength" files="com.azure.ai.textanalytics.implementation.TextAnalyticsClientImpl"/>
   <suppress checks="[a-zA-Z0-9]*" files="com.azure.data.schemaregistry.implementation.*"/>
+  <suppress checks="[a-zA-Z0-9]*" files="com.azure.opentelemetry.exporter.azuremonitor.implementation.*"/>
```

Comment:
Do you have an open issue for this? We shouldn't be suppressive every single checkstyle.